javascript/src/components/inputs/InputDropdown.jsx

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+var React = require('react/addons');
+var InputsStore = require('../../stores/inputs/InputsStore');
+var Input = require('react-bootstrap').Input;
+
+var InputDropdown = React.createClass({
+    mixins: [React.addons.LinkedStateMixin],
+    PLACEHOLDER: "placeholder",
+    getInitialState() {
+        return {
+            inputs: [],
+            selectedInput: this.PLACEHOLDER
+        };
+    },
+    loadData() {
+        InputsStore.list((inputs) => {
```

Comment:
This is loading a list of inputs, but a parent component (`javascript/src/components/messageloaders/LoaderTabs.jsx`) already loaded them, I think they should come inside a prop.